udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,10 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Uses async and await to grab content from the data servlet.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+async function getDataUsingAsyncAwait() {
```

Comment:
I'd recommend leaving "asyncAwait" out of the function name, since it's an implementation detail. In general, we strive to keep implementation details out of public API, since that makes it harder to change the API. For example, if the implementation is updated in the future to not use asyncAwait, then you'd need to change the function name, as well as references to the function.

Suggestion would be to name it like: `fetchCommentData()`, assuming that's what this is doing.

"Data" on its own is a bit generic, and could collide with other "get data" functions in the future.

Reply:
Done